README.md

```diff
@@ -33,6 +33,16 @@ See [https://creativecommons.org/licenses/by/3.0/][ccl] for more details.
 The following Google style guides live outside of this project:
 [Go Code Review Comments][go] and [Effective Dart][dart].
 
+
+## Contributing
+
+With few exceptions, these style guides are copies of Google's internal style guides
+to assist developers working on Google owned and originated open source projects.
+Changes to the style guides are made to the internal style guides first and
+eventually copied into the versions found here. **External contributions are
+not accepted.** Pull requests and issues filed in this repository may be
+closed without comment.
```

Comment:
I might expand this: Pull requests are regularly closed without comment. Some guides are modestly supported here: issues that raise questions and try to justify changes on technical merits may get some engagement (and could in theory lead to changes), but we are primarily optimizing for Google's internal needs.

Reply:
done